src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/AddRolesToExistingUserFuncTest.java

```diff
@@ -139,6 +142,7 @@ public void test_update_user_profile_with_roles_successfully_from_header() throw
     }
 
     @Test
+    @Ignore("Remove - not required")
     public void rdcc_418_ac1_should_update_user_status_from_active_to_suspended() throws Exception {

```

Comment:
instead of ignoring it.. you can remove it directly?

Reply:
Addressed here: https://github.com/hmcts/rd-user-profile-api/pull/621/commits/48b6d4cdb8805a197ca1ce639e325b06bd6be46b